src/main/java/seedu/address/ui/PatientCard.java

```diff
@@ -64,5 +73,22 @@ public PatientCard(Patient patient, int displayedIndex) {
         patient.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+
+        Set<Appointment> appointmentSet = patient.getAppointments();
+        appointmentsLabel.setText("Appointments: " + appointmentSet.size());
+
+        ObservableList<String> appointmentStrings = FXCollections.observableArrayList(appointmentSet.stream()
+                .map(appointment -> "Appointment " + appointment.getId() + ": With Dr. "
+                        + appointment.getDoctorName() + " on "
+                        + appointment.getDate().toString() + " (" + appointment.getTime().toString() + "hrs)")
+                .toArray(String[]::new));
```

Comment:
Great use of syntactic sugar to make code neater, more readable, and easier to understand:)